packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -7,6 +7,7 @@
 import { expect } from "chai";
 import { mount, ReactWrapper } from "enzyme";
 import * as React from "react";
+import * as sinon from "sinon";
```

Comment:
The sinon imports are inconsistent between packages. `core` imports sinon dependencies like this:

```tsx
import { Sinon, SinonSpy } from "sinon";
```

Whereas `datetime` imports everything as one module. Can we pick just one scheme?

Reply:
I started off by adding named sinon imports and later realized that I could have a much smaller diff if I switched to namespace imports in the other packages. I'm going to hold off on standardizing in this PR and leave it as a follow-up